chef/data_bags/crowbar/migrate/neutron/030_seperate_ml2_drivers.rb

```diff
@@ -0,0 +1,59 @@
+def upgrade ta, td, a, d
+  if a['networking_mode']
+    a['ml2_type_drivers'] = a['networking_mode']
+  else
+    a['ml2_type_drivers'] = ta['ml2_type_drivers']
+  end
+
+  if a['networking_plugin'] == 'linuxbridge'
+    a['networking_plugin'] = 'ml2'
+    a['ml2_mechanism_drivers'] = 'linuxbridge'
+  elsif a['networking_plugin'] == 'openvswitch'
+    a['networking_plugin'] = 'ml2'
+    a['ml2_mechanism_drivers'] = 'openvswitch'
+  elsif a['networking_plugin'] == 'cisco'
+    a['networking_plugin'] = 'ml2'
+    a['ml2_mechanism_drivers'] = 'cisco'
```

Comment:
The ml2 driver for cisco is called: ```cisco_nexus``` I wonder if we should use that name here as well.

And then there is the problem that this cannot be used standalone, it always needs be be used together with another mech driver, up to now we forced that to be ```openvswitch```. But ```linuxbridge``` might work as well.